AudioLibrary.WMME/WaveOutBuffer.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using AudioLibrary.WMME.Native;
+
+namespace AudioLibrary.WMME
+{
+    internal class WaveOutBuffer : IDisposable
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Fix this implementation of 'IDisposable' to conform to the dispose pattern. ([csharpsquid:S3881](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=csharpsquid%3AS3881))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-gu8DroMTOoScWu&open=AWfG1-gu8DroMTOoScWu)

[ISSUE_KEY]: AWfG1-gu8DroMTOoScWu